### ISSUE TITLE
Dependency resolution failed for Permit 0.2.0

### ISSUE THREAD

**Issue by sagastume** [curiosum-dev/permit_phoenix#14]
Hi,

I started a new project and installed versions 0.2.0 of Permit and Permit Ecto, but when I try to install the latest available version of permit_phoenix, it throws an error because permit_phoenix depends on Permit 0.1.3.

Are they not compatible anymore? Or is permit_phoenix no longer necessary?

mix.exs

````
{:permit, "~> 0.2.0"},
{:permit_ecto, "~> 0.2.0"},
{:permit_phoenix, "~> 0.1.0"}
````

Error:
````
Because permit_phoenix >= 0.1.0 depends on permit ~> 0.1.3 and your app depends on permit ~> 0.2.0, permit_phoenix >= 0.1.0 is forbidden.
So, because your app depends on permit_phoenix ~> 0.1.0, version solving failed.
** (Mix) Hex dependency resolution failed
````

**Comment by mennolpFoF** [comment]
@vincentvanbush Any idea when permit_phoenix will be bumped to v0.2.0 to solve this version mismatch?

**Comment by vincentvanbush** [comment]
@mennolpFoF I am working on it - will have it pushed today or tomorrow

**Comment by vincentvanbush** [comment]
@mennolpFoF Apologies for being late on this. We have released 0.2 versions as well as relevant patch updates to all libraries in question, and we're now writing down change announcements. Thanks for your patience!
